tests/functional/pv/pv_encryption/test_azure_kms_pv_encryption.py

```
@@ -97,22 +96,28 @@ def test_azure_kms_pv_encryption(
+                f"PV secret present in Azure KV for vol_handle={vol_handle}: "
+                f"expected=True, actual={self.kms.verify_pv_secrets_present_in_azure_kv(vol_handle)}"
+            )
             assert self.kms.verify_pv_secrets_present_in_azure_kv(
                 vol_handle
             ), f"PV secret for vol_handle : {vol_handle} not found in the Azure KV"
 
-        # pass
-
-        # Verify whether encrypted device is present inside the pod and run IO
+        logger.test_step("Verify encrypted devices and start IO on all pods")
         for vol_handle, pod_obj in zip(vol_handles, pod_objs):
```

Ruff (0.15.15)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)